frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualizationTask.scss

```diff
@@ -14,6 +14,9 @@ $border-color: var(--pf-global--BorderColor--light-100);
   &.is-linked {
     cursor: pointer;
   }
+  &--icon-spin {
+    transform-origin: 3.5% 14%;
+  }
```

Comment:
Sorry for the late reply. I was yesterday already confused by this "magic numbers" here. It looks like this are the % width and height of the parent container for the icon.

![image](https://user-images.githubusercontent.com/139310/106590421-5baec680-654d-11eb-9d28-b10b0135dc89.png)

You can set this also to the width and height (-0.1, don't know why :confused: ) if you want.

I tested this in the dev tools with this changes and it looks good for me.

PipelineVisualizationTask.scss:

```suggestion
  &--icon-spin {
    transform-origin: 16.61px 16.61px;
  }
```

When I also changed the viewBox in `PipelineVisualizationTask.tsx` to `viewBox="-5 -5 20 21"` to vertical align the Icon a little bit, I need to change this to this values:

```suggestion
  &--icon-spin {
    transform-origin: 16.61px 15.61px;
  }
```

Wdyt?

I tested this because I'm interessted into this magic number. If Andrew gives an approval here I'm also fine with that.

Reply:
Updated `transform-origin` to `16.61px`. I think values in `px` would be better than percentages. It's confusing, I thought that it would be calculated for the element that gets the animation, and not it's parent.
And `fa-spin` seems to animate simply using `rotate`.